packages/ketcher-core/src/domain/serializers/ket/ketSerializer.ts

```diff
@@ -318,21 +318,31 @@ export class KetSerializer implements Serializer<Struct> {
             .monomer as BaseMonomer;
           monomerIdsMap[node.$ref] = monomer?.id;
 
+          const { attachmentPointsList } =
+            BaseMonomer.getAttachmentPointDictFromMonomerDefinition(
+              template.attachmentPoints || [],
+            );
+          const parsedAttachmentPointsNumbers = [];
+
           template.attachmentPoints?.forEach(
             (attachmentPoint, attachmentPointIndex) => {
               const leavingGroupAtom = monomer.monomerItem.struct.atoms.get(
                 attachmentPoint.leavingGroup?.atoms[0] ||
                   attachmentPoint.attachmentAtom,
               );
               assert(leavingGroupAtom);
+              parsedAttachmentPointsNumbers.push();
```

Comment:
It seems like this line of code does nothing :)

Reply:
oops